### PR TITLE
Create CocoaPods spec

### PR DESCRIPTION
**Pull request by jondwillis**
Hey, I'm preparing to use this in a project and I'd love to see it in CocoaPods too, so I made a rough spec for this library.

### CODE REVIEW

**Comment by kastiglione** [comment]
Awesome, it's something I figured I would do, I guess now seems like a good time :fast_forward:. I'll give this spec a try today.

**Comment by kastiglione** [comment]
I haven't yet written wrappers for the entire Parse API. Let me know if there are parts you need added, or feel free to submit a PR.

**Comment by jondwillis** [comment]
@kastiglione Cool. My colleague actually created a library of the same name, but yours is more complete. I'm sure we will have some more contributions soon.

File: Parse-RACExtensions.podspec [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#
+# Be sure to run `pod spec lint Parse-RACExtensions.podspec' to ensure this is a
+# valid spec and remove all comments before submitting the spec.
+#
+# To learn more about the attributes see http://docs.cocoapods.org/specification.html
+#
+Pod::Spec.new do |s|
+  s.name         = "Parse-RACExtensions"
+  s.version      = "1.0.0"
```
Comment (kastiglione) [inline Parse-RACExtensions.podspec]: I've just tagged `v0.0.1`, so this'll need to change to `0.0.1`.

File: Parse-RACExtensions.podspec [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  s.summary      = "ReactiveCocoa extensions for Parse."
+  s.homepage     = "https://github.com/kastiglione/Parse-RACExtensions"
+  s.license      = { :type => 'MIT', :file => 'LICENSE.txt' }
+  s.author       = { "Dave Lee" => "dave@kastiglione.com" }
+  s.source       = { :git => "https://github.com/kastiglione/Parse-RACExtensions.git", :tag => "1.0.0" }
```
Comment (kastiglione) [inline Parse-RACExtensions.podspec]: As above, this will need to change to `v0.0.1`.

**Comment by kastiglione** [comment]
> My colleague actually created a library of the same name, but yours is more complete. I'm sure we will have some more contributions soon.

Sounds good. Let me know if you want me to add you as collaborators to the repo.

**Comment by jondwillis** [comment]
That'd be great. @mhupman @jondwillis

File: Parse-RACExtensions.podspec [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+# To learn more about the attributes see http://docs.cocoapods.org/specification.html
+#
+Pod::Spec.new do |s|
+  s.name         = "Parse-RACExtensions"
+  s.version      = "0.0.1"
+  s.summary      = "ReactiveCocoa extensions for Parse."
+  s.homepage     = "https://github.com/kastiglione/Parse-RACExtensions"
+  s.license      = { :type => 'MIT', :file => 'LICENSE.txt' }
+  s.author       = { "Dave Lee" => "dave@kastiglione.com" }
+  s.source       = { :git => "https://github.com/kastiglione/Parse-RACExtensions.git", :tag => "0.0.1" }
```
Comment (kastiglione) [inline Parse-RACExtensions.podspec]: Apologies for the inconsistency, but this one needs to be `v0.0.1` since I use a `v` prefix for version tags.

**Comment by kastiglione** [comment]
Alright, you're added. Generally, still do use pull requests rather than pushing directly.

**Comment by jondwillis** [comment]
Thanks. That's what I'm used to anyway!

**Comment by kastiglione** [comment]
:tangerine:

**Pull request merged**